gradle.properties

```diff
@@ -1,4 +1,5 @@
 VERSION_NAME=10.5.0-SNAPSHOT
+LAST_STABLE_VERSION_NAME=10.5.0
```

Comment:
link internal PR with instructions to update that value during the release? or could we even automate setting this property during the release idk